src/DebuggerData.h

```diff
@@ -67,10 +69,13 @@ class Breakpoints
 
 	std::vector<Breakpoint> breakpoints;
 	MemoryLayout* memLayout;
+	int currentIndex;
```

Comment:
Hmm, I'm confused.
The `currentIndex` member variable is only introduced in this commit. But prior commits already used that variable?
Does that mean the prior commits would not compile???
(Please make sure all commits compile without errors).

Reply:
No, I just reordered the commit by file instead of by time. So that you don't have to review old intermediary code.